README.md

```diff
@@ -49,6 +49,12 @@ You can also pass in the size you would like the widget to be:
 display(DeephavenWidget(t, width=100, height=250))
 ```
 
+### Alternate Deephaven Server URL
+By default, the Deephaven server is located at `http://localhost:{port}`, where `{port}` is the port set in the Deephaven server creation call. If the server is not there, such as when runnning a Jupyter notebook in a Docker container, modify the `DEEPHAVEN_IPY_URL` environmental variable to the correct URL before creating a `DeephavenWidget`. 
+```python
+import os 
+os.environ["DEEPHAVEN_IPY_URL"] = "http://localhost:1234"
```

Comment:
This is setting from within the python session, should also include instructions for settings IPY_URL from environment variable when launching jupyter.

Reply:
That's going to be highly dependent on environment. Do you mean in the case of a docker image?

Reply:
Ehh yea I guess this is fine. User should be able to figure out that they can set an Environment variable in other ways.